webapp/src/components/progress_bars/ProgressIcon.vue

```diff
@@ -11,7 +11,8 @@
           :class="props.active ? 'bg-slate-500' : 'bg-slate-200'" 
         >
           <div
-            class="py-1"
+            class="py-1 rounded"
+            :class="props.active ? 'bg-disco-blue' : 'bg-slate-200'"
           />
```

Comment:
```suggestion
          :class="props.active ? 'bg-disco-blue' : 'bg-slate-200'" 
        >
          <div class="py-1" />
```
This is equivalent and simpler imo